brickstrap.sh

```diff
@@ -668,9 +648,10 @@ function brp_run_command()
 
 function brp_run()
 {
-    brp_parse_cli_options "$@" && \
-    brp_validate_cli_options "$BRP_CMD" "$BRP_CMD_ARG" && \
-    brp_init_env && \
+    brp_image_drv_register_defaults
```

Comment:
Currently, brickstrap is exiting without an error message somewhere in this function.